### PR TITLE
Answer project-specific queries

### PR DESCRIPTION
**Pull request by rbowen**

### CODE REVIEW

File: wg-welcome/templates/project-specific-help.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+## Question
+
+Sometimes people post to dev@community with a question about a specific
+Apache project. We are not the place for that question, and we need to
+gently redirect, while offering a little additional information about
+how the ASF works.
+
+## Answer
+
+Hi, _____, thanks for your interest in Apache.
```
Comment (tisonkun) [inline wg-welcome/templates/project-specific-help.md:12]: Two comments here:

1. Maybe we can change the placeholder as parameters so such email can even be generated with parameters set.

File: wg-welcome/templates/project-specific-help.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+project has their own mailing lists, or other forums, for answering
+questions. This list - dev@community.apache.org - is about building
+communities across all these projects, but isn't usually the right place
+for project-specific questions.
+
+Your question is about the Apache [Foo] project, so the best place to seek
+advice is at https://[foo].apache.org/ or on their project-specific
+mailing lists at https://lists.apache.org/list.html?users@[foo].apache.org
+
+*** NOTE: Verify that the project in question has a users@ list, and
```
Comment (tisonkun) [inline wg-welcome/templates/project-specific-help.md:24]: 2. Is this NOTE for the reply sender? Or it's included AS IS in the reply for the receiver.

Comment (rbowen) [reply wg-welcome/templates/project-specific-help.md:24]: The note is for the sender. Ideally we would always edit these boilerplate messages, and customize them for each case.

**Review by tisonkun — APPROVED** [review]
Thanks for your information. I may pick up my comment 1 when I have more time :D

**Pull request merged**